packages/preview-middleware-client/test/unit/adp/init-dialogs.test.ts

```diff
@@ -1,18 +1,21 @@
 import type UI5Element from 'sap/ui/core/Element';
 import { RTAOptions } from 'sap/ui/rta/RuntimeAuthoring';
+import type ElementOverlay from 'sap/ui/dt/ElementOverlay';
 import type RuntimeAuthoring from 'sap/ui/rta/RuntimeAuthoring';
 
+import FlUtils from 'sap/ui/fl/Utils';
 import { sapMock } from 'mock/window';
+import Utils from 'mock/sap/ui/fl/Utils';
 import Fragment from 'mock/sap/ui/core/Fragment';
 import Controller from 'mock/sap/ui/core/mvc/Controller';
 import RuntimeAuthoringMock from 'mock/sap/ui/rta/RuntimeAuthoring';
 
-import { DialogNames, handler, initDialogs, isControllerExtensionEnabled } from '../../../src/adp/init-dialogs';
+import { DialogNames, handler, initDialogs, isFragmentCommandEnabled, isControllerExtensionEnabled } from '../../../src/adp/init-dialogs';
 import AddFragment from '../../../src/adp/controllers/AddFragment.controller';
 import ControllerExtension from '../../../src/adp/controllers/ControllerExtension.controller';
 import ExtensionPoint from '../../../src/adp/controllers/ExtensionPoint.controller';
 import ElementOverlay from 'sap/ui/dt/ElementOverlay';
-import FlUtils from 'sap/ui/fl/Utils';
+
```

Comment:
This is what probably causing lint errors.

Reply:
Hopefully that fixes it: [24b72be](https://github.com/SAP/open-ux-tools/pull/1905/commits/24b72bef8a7f14575a6b9aee8252c5d0c7aac7d6)